jre_emul/android/platform/libcore/ojluni/src/main/java/java/util/zip/ZipUtils.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (c) 2013, 2015, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Change the copyright to our Apache license, since you wrote it and not Oracle engineers. The easiest way to do so is to first move the class to com.google.j2objc.util (see below comment), and then copy the copyright comment from any of that package's source files.

Reply:
copied the copyright from other files in the same package